packages/client/src/components/SafeContacts.js

```diff
@@ -1,43 +1,191 @@
-import React from "react";
-import { Person } from "./Svg";
-import { useClipboard } from "../hooks";
+import React, { useState } from "react";
```

Comment:
Can we break down this file into the format of one component per file? It's easier to read and maintain.

Reply:
Personally I prefer to have code logic co-located until it gets too long so i'm not bouncing between files to piece logic together. If we want to enforce this paradigm though we should add https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-multi-comp.md to our eslint

Reply:
I understand it's easier to work on the single file, but I think we should still choose the one component per file paradigm for the following reason:
- It's difficult to determine and enforce the threshold, everyone has different standard in terms of how long is too long
- For me it's actually easier to piece logic together with separate files since I can split panel on my editor and do cross reference, if it's one single file I'm constantly scrolling back and forth.
- When we want to add unit test for components it's easier to organize if it's one component per file.